src/image-editor/ImageEditor.ts

```diff
@@ -49,7 +34,6 @@ export class ImageEditorWrapper {
           menuBarPosition: "bottom",
           usageStatistics: false,
         },
-        cssMaxWidth: 500,
         cssMaxHeight: 500,
```

Comment:
Why height still 500?

Reply:
I think it's better to keep this value to avoid vertical scrolling for the image editor


If remove this property and in the small screen
<img width="1388" alt="image" src="https://github.com/genkisystem/magic-point-sdk/assets/73162691/142353db-5198-4285-bf9c-4e0ad4e9fa34">